exec/csnex/Bytecode.cs

```diff
@@ -0,0 +1,144 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace csnex
+{
+    public class Bytecode
+    {
+        private byte[] obj;
+        public string source_path;
+        private byte[] source_hash;
+        private uint version;
+        public uint global_size;
+        public List<String> strtable;
+        private uint strtablesize;
+        public byte[] code;
+        public uint codelen;
+
+        private uint functionsize;
```

Comment:
This does not seem to need to be a class member.

Reply:
Changed to local variable.

Reply:
Removed.